### PR TITLE
fix(examples): make with-graphql-hooks run correctly again

### LINKED ISSUE #20474: Example with-graphql-hooks broken request
# Bug report

## Describe the bug
**Two issues:**

**Issue 1 - On creating example app, initial load is broken with error**. `Error serializing .initialGraphQLState["6gftu9"].data returned from getStaticProps in "/". Reason: undefined cannot be serialized as JSON. Please use null or omit this value.` This is from the `undefined` value in the return not being JSON parsable (see https://dev.to/ryyppy/reason-records-nextjs-undefined-and-getstaticprops-5d46). The request during this initial load of the example is returning a `502`.

**Issue 2 - Broken request to existing api using code in example.**  When utilizing the logic in the example into an existing minimal next.js boilerplate, `400 bad request` is continually returned throughout troubleshooting attempts.

```
error: {
    httpError: {
      status: 400,
      statusText: 'Bad Request',
      body: '{"errors":[{"message":"body.charCodeAt is not a function"}]}'
    }
  }
 ```
 
The api url is an existing app api that I can confirm the function in graphiql. Testing query is a minimal test example that is proven valid in the graphiql. 

Here are the request inputs:
```
query {
  kind: 'Document',
  definitions: [
    {
      kind: 'OperationDefinition',
      operation: 'query',
      name: [Object],
      variableDefinitions: [Array],
      directives: [],
      selectionSet: [Object]
    }
  ],
  loc: { start: 0, end: 128 }
}
operation {
  query: {
    kind: 'Document',
    definitions: [ [Object] ],
    loc: { start: 0, end: 128 }
  },
  variables: { count: 5 },
  operationName: undefined,
  persisted: undefined
}
opts { useCache: true, variables: { count: 5 } }
```

Here is the query (if it helps):
```
import gql from "graphql-tag";

export default gql`
  query getCreators($count: Int) {
    creators(first: $count) {
      nodes {
        name
      }
      totalCount
    }
  }
`;
```
 
All of the files used are as they are in the example repo. 

When watching my api logs, **no requests are hitting the api**

## To Reproduce

**Issue 1 - Broken initial code in example**
  1) npx create-next-app --example with-graphql-hooks with-graphql-hooks-app
  2) npm i
  3) npm run dev 
  4) view localhost:3000 to see error

**Issue 2 - Broken request to existing api using code in example**
  1) npx create-next-app --example with-graphql-hooks with-graphql-hooks-app
  2) npm i
  3) change url in graphql-client.js to existing api of your choice
  4) add query of your choice for api
  5) npm run dev 
  6) view localhost:3000 to see error

## Expected behavior

1) Expect example to return valid values and render example ui

2) Expect data to be present in api return for valid queries 

## System information

- OS: macOS Mojave 10.14.4 
- Chrome 87.0.4280.88
- Next.js: 10.0.0
- Node.js: 12.14.0

Merry Christmas! 🎅

### PR DESCRIPTION
**Pull request by ludofischer**
Fixes #20474

* Switch API url to working URL used in apollo example
* Update deps
* Move styled jsx out of the way to avoid demonstrating too many
  unfamiliar concepts at once

dev, build and start work now.

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| buildDuration | 10.4s | 10.6s | ⚠️ +151ms |
| nodeModulesSize | 80.8 MB | 80.8 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.061 | 2.064 | 0 |
| / avg req/sec | 1213.28 | 1211.44 | ⚠️ -1.84 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.313 | 1.349 | ⚠️ +0.04 |
| /error-in-render avg req/sec | 1904.66 | 1853.02 | ⚠️ -51.64 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| 677f882d2ed8..89aa.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-dba4ec1..27c0.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| polyfills-81..14d7.js gzip | 31.2 kB | 31.2 kB | ✓ |
| Overall change | 31.2 kB | 31.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _app-b6fc6bc..222c.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-e2ffa..0f3f.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-010c20..8411.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-d979103..c5ff.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| Overall change | 323 B | 323 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 619 B | 619 B | ✓ |
| withRouter.html gzip | 606 B | 606 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| buildDuration | 12.9s | 12.6s | -363ms |
| nodeModulesSize | 80.8 MB | 80.8 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| 677f882d2ed8..89aa.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-dba4ec1..27c0.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| polyfills-81..14d7.js gzip | 31.2 kB | 31.2 kB | ✓ |
| Overall change | 31.2 kB | 31.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _app-b6fc6bc..222c.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-e2ffa..0f3f.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-010c20..8411.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-d979103..c5ff.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| Overall change | 323 B | 323 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.18 MB | 5.18 MB | ✓ |

</details>

</details>
Commit: 475c9ce3c2a5d634680823e30f16af75e894fdbd

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| buildDuration | 11.3s | 11.3s | ⚠️ +8ms |
| nodeModulesSize | 80.9 MB | 80.9 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.216 | 2.262 | ⚠️ +0.05 |
| / avg req/sec | 1127.93 | 1105.44 | ⚠️ -22.49 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.453 | 1.445 | -0.01 |
| /error-in-render avg req/sec | 1720.92 | 1729.88 | +8.96 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| 677f882d2ed8..89aa.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-dba4ec1..27c0.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| polyfills-81..14d7.js gzip | 31.2 kB | 31.2 kB | ✓ |
| Overall change | 31.2 kB | 31.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _app-b6fc6bc..222c.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-e2ffa..0f3f.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-010c20..8411.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-d979103..c5ff.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| Overall change | 323 B | 323 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 619 B | 619 B | ✓ |
| withRouter.html gzip | 606 B | 606 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| buildDuration | 13.5s | 13.2s | -258ms |
| nodeModulesSize | 80.9 MB | 80.9 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| 677f882d2ed8..89aa.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-dba4ec1..27c0.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| polyfills-81..14d7.js gzip | 31.2 kB | 31.2 kB | ✓ |
| Overall change | 31.2 kB | 31.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _app-b6fc6bc..222c.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-e2ffa..0f3f.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-010c20..8411.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-d979103..c5ff.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| Overall change | 323 B | 323 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.18 MB | 5.18 MB | ✓ |

</details>

</details>
Commit: c12e466830fc907f3908d1f8fbc95d8d0ecfd849

File: examples/with-graphql-hooks/components/#app.js# [hunk 1]
```diff
@@ -0,0 +1,44 @@
+export default function App({ children }) {
```
Comment (timneutkens) [inline examples/with-graphql-hooks/components/#app.js#:1]: This file seems to be accidentally committed 🤔

Comment (ludofischer) [reply examples/with-graphql-hooks/components/#app.js#]: Removed it.

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| buildDuration | 10.3s | 9.9s | -340ms |
| nodeModulesSize | 80.8 MB | 80.9 MB | ⚠️ +10.1 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.03 | 2.016 | -0.01 |
| / avg req/sec | 1231.76 | 1239.78 | +8.02 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.283 | 1.249 | -0.03 |
| /error-in-render avg req/sec | 1949.19 | 2001.7 | +52.51 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| 677f882d2ed8..89aa.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-dba4ec1..27c0.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| polyfills-81..14d7.js gzip | 31.2 kB | 31.2 kB | ✓ |
| Overall change | 31.2 kB | 31.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _app-b6fc6bc..222c.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-e2ffa..0f3f.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-010c20..8411.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-d979103..c5ff.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| Overall change | 323 B | 323 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 619 B | 619 B | ✓ |
| withRouter.html gzip | 606 B | 606 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| buildDuration | 12s | 11.9s | -99ms |
| nodeModulesSize | 80.9 MB | 80.9 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| 677f882d2ed8..89aa.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-dba4ec1..27c0.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| polyfills-81..14d7.js gzip | 31.2 kB | 31.2 kB | ✓ |
| Overall change | 31.2 kB | 31.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _app-b6fc6bc..222c.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-e2ffa..0f3f.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-010c20..8411.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-d979103..c5ff.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| Overall change | 323 B | 323 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.18 MB | 5.18 MB | ✓ |

</details>

</details>
Commit: 12e93932a37f217371f9f657c9d403e323caab0e

**Review by timneutkens — APPROVED** [review]

**Comment by timneutkens** [comment]
Thanks!

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| buildDuration | 12.1s | 12.3s | ⚠️ +142ms |
| nodeModulesSize | 80.9 MB | 80.9 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.528 | 2.491 | -0.04 |
| / avg req/sec | 988.76 | 1003.62 | +14.86 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.697 | 1.771 | ⚠️ +0.07 |
| /error-in-render avg req/sec | 1472.81 | 1411.84 | ⚠️ -60.97 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| 677f882d2ed8..89aa.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-dba4ec1..27c0.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| polyfills-81..14d7.js gzip | 31.2 kB | 31.2 kB | ✓ |
| Overall change | 31.2 kB | 31.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _app-b6fc6bc..222c.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-e2ffa..0f3f.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-010c20..8411.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-d979103..c5ff.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| Overall change | 323 B | 323 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 619 B | 619 B | ✓ |
| withRouter.html gzip | 606 B | 606 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| buildDuration | 14.5s | 14.5s | -6ms |
| nodeModulesSize | 80.9 MB | 80.9 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| 677f882d2ed8..89aa.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-dba4ec1..27c0.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| polyfills-81..14d7.js gzip | 31.2 kB | 31.2 kB | ✓ |
| Overall change | 31.2 kB | 31.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _app-b6fc6bc..222c.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-e2ffa..0f3f.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-010c20..8411.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-d979103..c5ff.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| Overall change | 323 B | 323 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | ludofischer/next.js fix-with-graphql-hooks | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.18 MB | 5.18 MB | ✓ |

</details>

</details>
Commit: 9f1e57b8413a07180fee62bb6efd855cef013d6a

**Pull request merged**